examples/docker/echo_app.yaml

```diff
@@ -0,0 +1,27 @@
+# Runs the echo example app in a container with custom inputs and outputs
+#
+# The echo example ingests a file, prints the contents and writes it back out.
+# In this YAML, the output is mapped to a Sky Storage object, which writes to a
+# cloud bucket.
+#
+# Usage:
+#   sky launch -c myclus echo_app.yaml
+#   sky exec myclus echo_app.yaml
+#   sky down myclus
+
+file_mounts:
+  /inputs: ./echo_app
+  /echo_app: ./echo_app
```

Comment:
Can we change this to upload files instead, otherwise, we are uploading the folder twice, which may not be a good practice.

```
/inputs/README.md: ./echo_app/README.md
/echo_app: ./echo_app
```

Reply:
Good point, thanks!

Reply:
Turns out #1222 is blocker for this - how about we stick to mounting the whole folder?

Reply:
Sounds good. In that case, I think sticking to mounting the whole folder is fine for now.